clouds/aws/aws_roles/rhoam-sre-support-policy.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Do we want this file to be part of this repo?

Reply:
Will delete all rhoam related code from this repo (this file and [rhoam_add_role_policy.py](https://github.com/RedHatQE/cloud-tools/blob/962bbd0eaa28ce2cfdc01358dbb28adcc5483918/clouds/aws/aws_roles/rhoam_add_role_policy.py#L13)). 
The alternative location will be within the openshift-ci scenario files?
@rnetser @ILpinto

Reply:
no, should be in addon isntall - (ocp-addons-operators-cli)

Reply:
Removed.